aiida_registry/__init__.py

```diff
@@ -26,39 +26,49 @@
 entrypoint_metainfo = {
     'aiida.calculations': {
         'shortname': "Calculations",
-        'longname': "Calculation plugins",
+        'longname': "CalcJobs and CalcFunctions",
```

Comment:
note that the term `CalcFunction` actually does not appear anywhere in the documentation. The decorator is `calcfunction` and the decorated function is referred to as a "calculation function". Same for `WorkFunction` -> "work functions"

Reply:
thanks - fixed!